tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/fix_unmapped_survey_answers_test.py

```diff
@@ -0,0 +1,184 @@
+                (4088548,'Seen by general practitioner','Observation','SNOMED','Clinical Finding','S','185278000','1970-01-01','2099-12-31',null),
+                (1586151,'Asian: Asian Specific','Observation','PPI','Question','S','Asian_AsianSpecific','2017-05-17','2099-12-31',null),
+                (1333079,'Moderately unhappy','Observation','PPI','Answer','S','mhwb_a_40','2020-05-07','2099-12-31',null),
+                (9580,'month','Unit','UCUM','Unit','S','mo','1970-01-01','2099-12-31',null),
+                (1332724,'Months','Observation','PPI','Answer',null,'cope_a_198','2020-05-07','2099-12-31',null)
+        """)
+
+        insert_concept_query = concept_empl.render(
+            fq_dataset_name=f'{self.project_id}.{self.dataset_id}')
+
+        concept_relationship_empl = self.jinja_env.from_string("""
+                 INSERT INTO `{{fq_dataset_name}}.concept_relationship`
+                    (concept_id_1,concept_id_2,relationship_id,valid_start_date,valid_end_date,invalid_reason)
+                    VALUES
+                        (1332776,1333079,'Maps to','2020-05-07','2099-12-31',null),
+                        (1384515,4179221,'Maps to','2019-04-21','2099-12-31',null),
+                        (1384615,1384615,'Maps to','2019-04-21','2099-12-31',null),
+                        (43530243,4088548,'Maps to','2019-04-21','2099-12-31',null),
+                        (1585609,1586151,'Maps to','2019-04-21','2099-12-31',null),
+                        (1332724,9580,'Maps to','2020-05-07','2099-12-31',null)
+                """)
+
+        insert_concept_relationship_query = concept_relationship_empl.render(
+            fq_dataset_name=f'{self.project_id}.{self.dataset_id}')
+
+        observation_tmpl = self.jinja_env.from_string("""
+            INSERT INTO `{{fq_dataset_name}}.observation`
+            (observation_id, person_id, observation_concept_id, observation_date, 
+             observation_type_concept_id, value_as_concept_id, value_source_concept_id)
+            VALUES
+                (1, 111111, 1384592, '2015-07-15', 45905771, 1384615, 1384615),
+                (2, 222222, 1384592, '2015-07-15', 45905771, 0, 1384615),
+                (3, 333333, 1332749, '2015-07-15', 45905771, 0, 1332776),
+                (4, 444444, 43528660, '2015-07-15', 45905771, 0, 43530243),
+                (5, 555555, 1384487, '2015-07-15', 45905771, 0, 1384515), 
+                (6, 666666, 1332756, '2015-07-15', 45905771, 0, 1332724),
+                (7, 777777, 1586151, '2015-07-15', 45905771, 0, 1585609), 
+                (8, 888888, 0, '2015-07-15', 45905771, 0, 1585536)
+                """)
+        insert_observation_query = observation_tmpl.render(
+            fq_dataset_name=f'{self.project_id}.{self.dataset_id}')
+
+        # Load test data
+        self.load_test_data([
+            insert_observation_query, insert_concept_query,
+            insert_concept_relationship_query
+        ])
```

Comment:
Executing a single bigquery script would be a lot more efficient:
```suggestion
        self.load_test_data([
            f'''{insert_observation_query};
                {insert_concept_query};
                {insert_concept_relationship_query}'''
        ])
```